README.md

```diff
@@ -26,13 +26,25 @@
 
 The friendly Operating System for IoT!
 
+RIOT is a real-time multi-threading operating system that supports a range of
+devices that are typically found in the Internet of Things (IoT): 
+8-bit microcontrollers, 16-bit microcontrollers and light-weight 32-bit processors.
+
+RIOT is based on the following design principles: energy-efficiency, real-time
+capabilities, small memory footprint, modularity, and uniform API access,
+independent of the underlying hardware (this API offers partial POSIX
+compliance).
+
+RIOT is developed by an international open source community which is
+independent of specific vendors (e.g. similarly to the Linux community)
+and is licensed with a non-viral copyleft license (LGPLv2.1), which allows
```

Comment:
I somehow dislike the term "non-viral":
1.) I'm not sure if and how LGPL qualifies
2.) this suggests that other copyleft licenses are "viral" which has a negative connotation.